app/language/language-core/src/main/kotlin/com/hedvig/android/language/Language.kt

```diff
@@ -47,20 +53,30 @@ enum class Language {
     const val BCP_47_EN_NO = "en-NO"
     const val BCP_47_DA_DK = "da-DK"
     const val BCP_47_EN_DK = "en-DK"
+    const val BCP_47_SV = "sv"
+    const val BCP_47_EN = "en"
+    const val BCP_47_SV_FOR_SUBSTRING = "sv-"
+    const val BCP_47_EN_FOR_SUBSTRING = "en-"
 
     /**
      * Parses the language tag, in BCP-47 format, to [Language]
      * See [RFC-5646](https://www.rfc-editor.org/info/rfc5646) for more information
      * @param value: A language tag in BCP-47 format
      */
-    fun from(value: String): Language = when (value) {
-      BCP_47_SV_SE -> SV_SE
-      BCP_47_EN_SE -> EN_SE
-      BCP_47_NB_NO -> NB_NO
-      BCP_47_EN_NO -> EN_NO
-      BCP_47_DA_DK -> DA_DK
-      BCP_47_EN_DK -> EN_DK
+    fun from(value: String): Language = when {
+      value == BCP_47_SV_SE -> SV_SE
+      value == BCP_47_EN_SE -> EN_SE
+      value == BCP_47_NB_NO -> NB_NO
+      value == BCP_47_EN_NO -> EN_NO
+      value == BCP_47_DA_DK -> DA_DK
+      value == BCP_47_EN_DK -> EN_DK
+      value == BCP_47_EN || value.startsWithSubstring(BCP_47_EN_FOR_SUBSTRING) -> EN_GLOBAL
+      value == BCP_47_SV || value.startsWithSubstring(BCP_47_SV_FOR_SUBSTRING) -> SV_GLOBAL
```

Comment:
I see yeah, I think this is a great idea, let's see how this fares!
Should we also add a log over here to catch the cases when this happens, to see what other values we're getting and how often? I am mostly curious, and we can then see if it aligns with the people who have had issues with this.